### PR TITLE
Reward was added to Account stats

### PR DESCRIPTION
**Pull request by ahmedrns**
putAccountStats method was updated
Some variable updated in proposal API

### CODE REVIEW

File: src/Types.ts [hunk 1]
```diff
@@ -795,6 +795,11 @@ export interface IAccountInformation {
      */
     total_frozen: JSBI;
 
+    /**
+     * Total frozen amount of account
```
Comment (TrustHenry) [inline src/Types.ts]: The document is invalid. (copy and paste)

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -1355,16 +1355,31 @@ export class LedgerStorage extends Storages {
...
+            let accountInfo: IAccountInformation = {
+                total_balance: JSBI.BigInt(0),
+                total_spendable: JSBI.BigInt(0),
+                total_frozen: JSBI.BigInt(0),
+                total_reward: JSBI.BigInt(0),
                 tx_count: total_txs[0].tx_count ? total_txs[0].tx_count : 0,
             };
+            utxo_array.map((elem) => {
+                switch (elem.type) {
+                    case 0: {
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: OutputType.Payment

File: src/modules/storage/LedgerStorage.ts [hunk 3]
```diff
@@ -1355,16 +1355,31 @@ export class LedgerStorage extends Storages {
...
+                        accountInfo.total_spendable = JSBI.add(accountInfo.total_spendable, elem.amount.value);
+                        break;
+                    }
+                    case 1: {
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: OutputType.Freeze

File: src/modules/storage/LedgerStorage.ts [hunk 4]
```diff
@@ -1355,16 +1355,31 @@ export class LedgerStorage extends Storages {
...
+                        accountInfo.total_frozen = JSBI.add(accountInfo.total_frozen, elem.amount.value);
+                        break;
+                    }
+                    default: {
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: OutputType.Coinbase

Comment (TrustHenry) [reply src/modules/storage/LedgerStorage.ts]: And default

**Review by TrustHenry — CHANGES_REQUESTED** [review]

File: src/modules/storage/LedgerStorage.ts (hunk 4 above)
Comment (ahmedrns) [reply src/modules/storage/LedgerStorage.ts]: resolved

File: src/modules/storage/LedgerStorage.ts (hunk 3 above)
Comment (ahmedrns) [reply src/modules/storage/LedgerStorage.ts]: resolved

File: src/modules/storage/LedgerStorage.ts (hunk 2 above)
Comment (ahmedrns) [reply src/modules/storage/LedgerStorage.ts]: resolved

File: src/Types.ts (hunk 1 above)
Comment (ahmedrns) [reply src/Types.ts]: resolved

File: src/modules/storage/LedgerStorage.ts [hunk 5]
```diff
@@ -1355,16 +1355,34 @@ export class LedgerStorage extends Storages {
...
+                switch (elem.type) {
+                    case OutputType.Payment: {
+                        accountInfo.total_spendable = JSBI.add(accountInfo.total_spendable, elem.amount.value);
+                        break;
+                    }
+                    case OutputType.Freeze: {
+                        accountInfo.total_frozen = JSBI.add(accountInfo.total_frozen, elem.amount.value);
+                        break;
+                    }
+                    case OutputType.Coinbase: {
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:1376]: Coinbase transactions have empty sender(Inputs).

Comment (hewison-chris) [reply src/modules/storage/LedgerStorage.ts:1376]: For `Coinbase` transactions the single input is `hashFull(height)`

Comment (ahmedrns) [reply src/modules/storage/LedgerStorage.ts:1376]: when there's no senders, we do not deal with senders array

Comment (ahmedrns) [reply src/modules/storage/LedgerStorage.ts:1376]: transaction at block height 0 has no senders, so that only receivers array is traversed

Comment (TrustHenry) [reply src/modules/storage/LedgerStorage.ts:1376]: Yes, I checked it.
https://v7.bosagora.io/blocks_from?height=288&max_blocks=1

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**